addons/mail/static/src/views/web/fields/emojis_field_common/emojis_field_common.xml

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-
 <templates>
     <t t-name="mail.EmojisFieldButton">
         <div class="o_mail_emojis_buttons position-relative d-inline">
             <button
-                class="btn end-0"
+                class="btn position-absolute end-0"
```

Comment:
Not sure this one changes anything, the div is probably relatively small (?)

Reply:
Without 
![image](https://github.com/user-attachments/assets/145d1a19-725a-41c7-9b51-accad67f2878)

Reply:
@jaeschwa column direction should only apply to the text field, not char fields.

Alternatively, make the div not be position-relative so that the absolute is relative to the field widget. Otherwise it's pointless to make it relative.